lib/views/dishes.erb

```diff
@@ -0,0 +1,12 @@
+Main Courses
+<% @dishes.each do |dish| %>
+<form class="form" action="index.html" method="post">
+<%= dish.name %>: $<%= dish.price %>
+  <input type="button" name="addto" id="addto" value="Add to" />
+<br />
+<% end %>
+</form>
+
+<form class="form" action="index.html" method="post">
+  <input type="button" name="addto" id="addto" value="CHECKOUT" />
```

Comment:
You are going to run into problems later if you make both the "Add to" button and the "CHECKOUT" button have the same id.